### DISCUSSION TITLE
Fooocus - Image Sharpness in WebUI AUTOMATIC1111

### DISCUSSION THREAD

**Discussion by edusachs** [lllyasviel/Fooocus#2786] [Q&A]
Hello everyone!

I'm currently working with Fooocus and the AUTOMATIC1111 WebUI. A question that came up was related to the "Image Sharpness" option in Fooocus. Changing this setting from 2 to 11, I noticed a change in the image colors, which seem to gain more contrast (I'm not exactly sure what it is), as you can see in the image I attached. It seems that the image gains more life!

It seems that adjusting the Image Sharpness helps to intensify the colors, which is particularly useful in images of people. However, I'm trying to find a way to apply a similar adjustment in the AUTOMATIC1111 WebUI. Does anyone have experience with this or know how I can add this functionality?

Thanks in advance!
![sharpness-2-11](https://github.com/lllyasviel/Fooocus/assets/34560360/297d1c1b-20d3-45a8-88a7-2794098d5aa8)

**Comment by mashb1t** [comment]
You can partially get a similar effect by increasing the CFG scale, but you need additional code changes in A1111 to get the exact same effect.
See https://github.com/lllyasviel/Fooocus/blob/main/modules/patch.py#L245

**Reply by edusachs** [reply]
Thank you @mashb1t

**Comment by DavidDragonsage** [comment]
I have found that the Detail Tweaker XL LoRA gives similar results to the Fooocus Image Sharpness control:
https://civitai.com/models/122359?modelVersionId=135867

However, I think that the Fooocus Image Sharpness function does a better job and I never use Detail Tweaker with Fooocus.

**Reply by edusachs** [reply]
I'll try it, thanks!
